sorc/link_workflow.sh

```diff
@@ -28,6 +28,7 @@ SLINK="ln -fs"
 pwd=$(pwd -P)
 
 # Link post
+[[ -d upp.fd ]] && rm -rf upp.fd
```

Comment:
But since it links to a dir it should return `0` still

Reply:
You are right.  I was just following the pattern already in `link_workflow.sh`.   If we replace `-d` with `-L` or `-h`, it should be done consistently throughout `link_workflow.sh`.   Repo code managers should review and update the sections of `link_workflow.sh` as directed by the g-w team.   What's the g-w team's preference:  `-L` or `-h` for symlinks?

Reply:
You can leave it with `-d` for now.  I will make a mental note of it and save it for later.

Reply:
I tested `-L` and it works as intended.   This test reminded me that NCO's execution of `link_workflow.sh` sets `LINK="cp -rp"`.    Thus, for NCO, 

```
# Link post
[[ -d upp.fd ]] && rm -rf upp.fd
$LINK ufs_model.fd/FV3/upp upp.fd
```
results in a physical directory named `upp.fd`   In this case `-d` is the proper test.   If we use `-h` or `-L`, the test is `false` and the directory is not removed.   This may not matter so much since the next line does a `cp -rp`.   For clarity and consistency, it would be good for the directory to be removed before `ufs_model.fd/FV3/upp` is copied.

I'm not sure how we want to handle the `ln -fs` versus `cp -rp` differences in `LINK` depending on the `RUN_ENVIR`.

One idea.   `link_workflow.sh` includes both `LINK` and `SLINK`.    `-h` or `-L` checks are appropriate for `SLINK`.   `-d` works for `LINK`.   We could review the use of `LINK` and `SLINK` in `link_workflow.sh` and update as appropriate.   Once this is done, a second sweep through `link_workflow.sh` could ensure `-d` is used for `LINK` directories and `-h` or `_L` is used for `SLINK` directories.   This or related development should be the subject of a new issue and subsequent PR.

Reply:
Keep in mind when NCO runs these, they make copies instead of symlinks.